src/views/LoginView.vue

```diff
@@ -1,114 +1,196 @@
+            class="text-h6"
+            @click ="$refs.usernameInput.focus()"
+          >
+            Sign up 
+            <img class="ml-2" src="/img/loginView/icon-account.svg" />
+          </v-btn>        
+        </nav>
+
+        <div class="d-flex h-75 flex-column justify-space-around align-center mt-2">
+          <h1>Welcome to Cuttle.Cards</h1>
+          <p class="text-h4 mb-4">The card game that will have you hooked from the first hand!</p>
+        
+          <div class="video-container__wrapper">
+            <div class="video-container">
+              <iframe
+                class="video-container__video"
+                src="https://www.youtube.com/embed/qOqkNbhMdsI"
+                title="Cuttle Game Tutorial -- Youtube Player"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen
+              />
+            </div>
+            <!-- Grouped for the ease of styling -->
             <v-btn
-              :loading="loading"
-              color="primary"
-              block
-              type="submit"
-              data-cy="submit"
-            >
-              {{ buttonText }}
-            </v-btn>
+              class="w-100 mt-8"
+              size='x-large'
+              to="/rules"
+              color="newPrimary"
+              data-cy="rules-link"
+            >Read The Rules</v-btn>
           </div>
-        </form>
 
-        <div id="switch-button-container">
-          <v-btn
-            color="primary"
-            variant="text"
-            data-cy="switch-mode"
-            @click="switchMode"
-          >
-            {{ switchLabelText }}
-          </v-btn>
-        </div>
-
-        <BaseSnackbar
-          v-model="showSnackBar"
-          :message="snackBarMessage"
-          color="error"
-          data-cy="auth-snackbar"
-          @clear="clearSnackBar"
-        />
-
-      </v-col>
-    </v-row>
-
-    <v-row class="mt-8">
-      <h1 class="gradient-text">What is Cuttle?</h1>
-    </v-row>
-
-    <v-row class="mt-0">
-      <v-col :cols="$vuetify.display.mdAndUp ? 6 : 12" class="d-flex justify-start flex-column mt-4">
-        <blockquote class="quote">
+        </div>    
+      </v-container>
+    </section>
+
+    <!--////////////////////////// 
+     /////Login Section /////// 
+    //////////////////////////-->
+
+    <section>
+      <v-container id="login-container" class="container">
+        <img id="logo" alt="Cuttle logo" src="/img/logo.png" />
+        <v-row>
+          <!-- Left side form -->
+          <v-col id="username-login-form" :cols="$vuetify.display.mdAndUp ? 8 : 12" >
+            <p class="text-h4 text-center font-weight-medium mb-10 ">
+              {{ formHeaderText }}
+            </p>
+
+            <v-form
+              v-model="isFormValid"
+              id="login"
+              @submit.prevent="submitLogin"
+              class="mx-auto"
```

Comment:
per the [vue style guide](https://v2.vuejs.org/v2/style-guide/?redirect=true#Element-attribute-order-recommended) it's nicer to put the id before v-model before the other stuff, before the @ handlers, which go last
```suggestion
              id="login"
              v-model="isFormValid"
              class="mx-auto"
              @submit.prevent="submitLogin"
```

Reply:
This is a great case to create a ticket  for an eslint improvement to force an order on component props/attrs.

Reply:
@seriouslysean I created #459 for this